giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/config/impl/UserHelper.java

```diff
@@ -49,4 +61,13 @@ public User createUser(Connection<?> connection) {
     public String createUsername(String username, String providerId) {
         return username + "_" + providerId;
     }
+    
+    @Override
+    public boolean checkUserPermission(IDocument document, CitesphereToken citesphereToken) {
```

Comment:
since the method return a boolean, the method name should be `is...` or `has...`.